packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -605,6 +605,60 @@ export default class RestApi {
       res.redirect(308, req.baseUrl + '/backup/jobs/vm/' + req.params.id)
     })
 
+    api.get(
+      '/dashboard',
+      wrap(async (req, res) => {
```

Comment:
I believe we need to add a debounce for this call (I believe we can start with 1 min).

Reply:
`debounce`?
Do you mean `throttle` instead?

Reply:
Throttle if you want, I mean that the result should be cached for a minute.